packages/framework-info/README.md

```diff
@@ -0,0 +1,381 @@
+[![npm version](https://img.shields.io/npm/v/@netlify/framework-info.svg)](https://npmjs.org/package/@netlify/framework-info)
+[![Coverage Status](https://codecov.io/gh/netlify/framework-info/branch/main/graph/badge.svg)](https://codecov.io/gh/netlify/framework-info)
+[![Build](https://github.com/netlify/framework-info/workflows/Build/badge.svg)](https://github.com/netlify/framework-info/actions)
+[![Downloads](https://img.shields.io/npm/dm/@netlify/framework-info.svg)](https://www.npmjs.com/package/@netlify/framework-info)
```

Comment:
I think they need to be updated

Reply:
Would you be open to having these just removed outright? Personally I'm not sure how useful they are as statuses here

Reply:
let's remove them :)